src/Components/QuestionsDisplay.js

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react'
+
+const QuestionsDisplay = (props) => {
+    const {
+        questionNumber,
+        data,
+        handleUserAnswerSelection,
+        randomWord,
+        wordOneChecked,
+        wordTwoChecked,
+        handleChange,
+        handleNextQuestion,
+        userAnswer,
+    } = props;
+
+    const [score, setScore] = useState(0);
+
+    let homophone;
+
+    const correctAnswerChecker = () => {
+        // Logic to check if answer is correct or not
+        console.log("user answer", userAnswer);
+        console.log("API answer", homophone);
+        
+        if (userAnswer === "") {
+            console.log("User Answer is Empty");
+        } else if (userAnswer === homophone) {
+            console.log("Correct");
+            setScore((prevState) => {
+                return prevState + 1;
+            })
+        } else {
+            console.log("Incorrect");
+        }
+    }
+
+    useEffect(() => {
+        correctAnswerChecker();
+    }, [userAnswer])
+
+    return (
+        <div className="App">
+            <h2>Question {questionNumber}:</h2>
+            <p>Score: {score} </p>
```

Comment:
Probably will want a score state at the App.js level so that it can be updated in the header